pkg/sql/logictest/testdata/logic_test/sequences

```diff
@@ -1084,3 +1084,135 @@ DROP SEQUENCE seq_50649
 
 statement ok
 DROP TABLE t_50649
+
+subtest regression_50712
```

Comment:
Kind of confusing how tests from a different commit ended up in this one, but I don't know if it's worth the trouble to fix.